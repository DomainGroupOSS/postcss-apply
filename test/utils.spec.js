import kebabify from '../src/utils';


describe('kekabify', () => {
  it('should convert camelCase properties to kebab-case', () => {
    const input = [
      'borderRadius',
      'backgroundColor',
      'background-color',
      'fontSize',
      'font-size',
      'borderTopLeftRadius',
      '-moz-Whatever',
      '-webkit-whoCares',
      '--customProp',
      '--CustomProp',
      '--custom-prop',
    ];

    const result = input.map(kebabify);

    expect(result).toMatchSnapshot();
  });
});
