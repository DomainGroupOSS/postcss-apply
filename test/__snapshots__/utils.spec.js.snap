// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kekabify should convert camelCase properties to kebab-case 1`] = `
Array [
  "border-radius",
  "background-color",
  "background-color",
  "font-size",
  "font-size",
  "border-top-left-radius",
  "-moz-whatever",
  "-webkit-who-cares",
  "--custom-prop",
  "--custom-prop",
  "--custom-prop",
]
`;
